paper/ms.tex

```diff
@@ -1557,10 +1569,10 @@ \subsubsection{The lifetime constraint}
 over the first $\approx$$10^8$\,yr; prominences might even be expected
 to be {\it more} common at the age of the Pleiades than for younger
 stars.  This broadly assumes that any star that can support
-prominences will, and that the magnetic topology of rapidly rotating M
+prominences will do so, and that the magnetic topology of rapidly rotating M
 dwarfs at 100\,Myr is similar to that of rapidly rotating M dwarfs at
 say, 10\,Myr.  Based on the degree to which M dwarfs remain rapid
-rotators over the first few gigayears of their lives
+rotators over the first few Gyr of their lives
```

Comment:
Reverting to "gigayears" - I don't think the "Myr" and "Gyr" abbreviations are all that widely known in astro

Reply:
I guess, but I think people could also pick it up from context clues :) Fairly minor though, I'm agnostic